envbuilder.go

```diff
@@ -394,6 +396,14 @@ func Run(ctx context.Context, options Options) error {
 		})
 	}
 
+	// temp move of all ro mounts
+	tempRemountDest := filepath.Join("/", MagicDir)
```

Comment:
Not introduced by your PR but I want to point out that `MagicDir` is kinda unhelpful as a name; perhaps `EnvbuilderWorkingDir` would be more explicit?

Reply:
https://github.com/coder/envbuilder/issues/184

Reply:
I'm not sure prefixing with envbuilder is that clarifying, just considering it's in the envbuilder repo.

I'm happy to change it from MagicDir though.

Reply:
`WorkingDir` SGTM too 👍